app/src/main/java/solendsdkfacade/model/SolendEnvironment.kt

```diff
@@ -0,0 +1,5 @@
+package solendsdkfacade.model
+
+enum class SolendEnvironment(val sdkValue: String) {
```

Comment:
in future - we can switch SolendEnvironment depending on BuildConfig or Debug panel